src/shared/containers/Refile/RefileErrorsForm.jsx

```diff
@@ -117,14 +133,64 @@ class RefileErrorsForm extends Component {
   }
 
   /**
-  * @desc Handles sending the form field payload from the state to the proper API endpoint. All
-  * fields are validated prior to executing the ajax call. Updates the form state booleans and result
-  * based on successful or error responses
-  * @param {object} event - contains the current event context of the field
+  * @desc When the submit button is clicked, it handles the event and executes handleFormSubmit()
+  * @param {event} event - the click event
+  */
+  clickSubmit(event) {
+    event.preventDefault();
+    this.handleFormSubmit(true);
+  }
+
+  /**
+  * @desc When the pagination button is clicked, it handles the event and executes
+  * handleFormSubmit()
+  * @param {string} type - indicates if it is previous button or next button has been clicked
   */
-  handleFormSubmit(event) {
+  hitPageButton(type) {
     event.preventDefault();
 
+    const resultLimit = this.state.formFields.resultLimit;
+    const pageIncrement = 1;
+    const offsetInt = parseInt(this.state.formFields.offset, 10);
+
+    // Set new state values based on the results from the new request
+    const setStateForPagination = () => {
+      const pageDifference = (type === 'pre') ? -1 : 1;
+
+      this.setState({
+        formFields: {
+          startDate: this.state.formFields.startDate,
+          endDate: this.state.formFields.endDate,
+          offset: this.state.formFields.offset + resultLimit * pageDifference,
+          resultLimit,
+        },
+        pageOfRefileErrorResults:
+          this.state.pageOfRefileErrorResults + pageIncrement * pageDifference,
+      }, () => { this.handleFormSubmit(); });
+    };
+
+    if (type === 'pre') {
+      if (offsetInt <= 0) {
+        return;
+      }
+
+      setStateForPagination();
```

Comment:
As this is the last thing in the `else` of this conditional too, could just move this line out of the conditional? (to (what is currently) line 185)

Reply:
I see what you are saying, this function has updated in the later code, so the logic is different. You can see the source of the last PR.
from line 170
https://github.com/NYPL/nypl-recap-admin/blob/add-two-more-columns-for-refile-error-display/src/shared/containers/Refile/RefileErrorsForm.jsx